src/main/java/stelnet/commodity/data/content/sell/DemandPrice.java

```diff
@@ -1,8 +1,9 @@
-package stelnet.commodity.data;
+package stelnet.commodity.data.content.sell;
```

Comment:
I could understand going one depth, but two? Perhaps just `content` will do? Class names are pretty explanatory.

Reply:
We can keep it as it is. This will definitely break save games as well.

http://www.javapractices.com/topic/TopicAction.do?Id=205

As I read through the project file tree, I get an understanding which classes belong together. For example "Sell information" belongs together with "DemandPrice" not with "SupplyPrice" so technically it is irrelevant info when I want to understand "what make sell information tick?".

When I made this PR I was just organizing code in a way that I can understand it better.

Reply:
Looks good to me. I did read this last year when I decided on packaging or original mods (stellics, hypernet, galmart), but forgot about this. 

I'm pretty sure the commodity will change a bit either way (and make mod save-incompatibile) so it's fine to change this now.